validate_sptx/cli.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import json
+import sys
+
+from starfish.util.argparse import FsExistsType
+from validate_sptx.validate_sptx import validate
+
+
+class Cli:
+    parser_group = None
+
+    @staticmethod
+    def add_to_parser(parser):
```

Comment:
Add short (potentially one-line) docstrings explaining what this does. Also for `run()` below.